.github/workflows/ci.yml

```diff
@@ -23,3 +23,6 @@ jobs:
           signingKey: "${{ secrets.CACHIX_SIGNING_KEY }}"
           authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
       - run: nix-build -A env
+      - run: which terraform
+      - name: Check terraform init 
+        run: nix-shell default.nix -A env --command "cd terraform/environment && terraform init --backend=false"
```

Comment:
Our `default.nix` does not work with `nix-shell` (@flokli doesn't like nix-shell. it's meant to interactively debug derivations; not for setting up `PATH`). 

Instead what you can do is:

```
export PATH=$(nix-build -A env)/bin:$PATH
( cd terraform/environment; terraform init --backend=false)
```

This is copying what our `.envrc` does in shell; basically
https://github.com/wireapp/wire-server-deploy/blob/develop/.envrc#L3